src/cmd-buildextend-metal

```diff
@@ -118,26 +113,43 @@ if [ "${rev_parsed}" != "${commit}" ]; then
     fi # otherwise, the repo already has a ref, so no need to create
 fi
 
+
+itype=metal
+img=${name}-${build}-${itype}.raw
+if [ -f "${builddir}/${img}" ]; then
+    echo "Bare metal image already exists"
+    exit
+fi
+
+path=${PWD}/${img}
 # for anaconda logs
 mkdir -p tmp/anaconda
 
-for itype in ${BIOS:+metal-bios} ${UEFI:+metal-uefi}; do
-    img=${name}-${build}-${itype}.raw
-    if [ -f "${builddir}/${img}" ]; then
-        echo "Image $itype already exists"
-        continue
-    fi
-
-    path=${PWD}/${img}
-    run_virtinstall "${ostree_repo}" "${ref}" "${path}.tmp" --variant="${itype}"
+if [ "$arch" == "aarch64" ]; then
+    run_virtinstall "${ostree_repo}" "${ref}" "${path}.tmp" --variant="${itype}-uefi"
```

Comment:
Hmm, how come we don't have to estimate the disk size in this path?

Reply:
We do, it's in run_virtinstall